iron/src/app/core-components/modal-container/modal-container.component.ts

```diff
@@ -0,0 +1,29 @@
+import { ChangeDetectionStrategy, Component, inject, input } from '@angular/core';
+import { ModalService } from '../../services/Modal.service';
+
+@Component({
+  selector: 'ls-modal-container',
+  standalone: true,
+  imports: [],
+  template: `
+  
+  <section class="modal">
+    <header>
+      <h1>{{ title() }}</h1>
+      <button (click)="closeDialoge()" type="button">X</button>
```

Comment:
Glaube Icon würde besser aussehen das X in der Figma war ja nur, weil ich kb auf Icon Suche hatte.

Reply:
Same halt